src/stylesheets/filter.scss

```diff
@@ -1,3 +1,17 @@
+.libraries-page > .panel:last-of-type {
```

Comment:
This would make more sense if it went in a `libraries_page.scss` file or maybe target the `.filter`'s Panel parent?